src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -1042,6 +1052,21 @@ public ColumnStatisticsTextModel retrieveColumnStats(
     return validateResponse(response, "dataset column stats", HttpStatus.OK, null);
   }
 
+  private DataRepoResponse<ColumnStatisticsIntModel> retrieveColumnStatsIntRaw(
+      TestConfiguration.User user, UUID datasetId, String table, String columnName, String filter)
+      throws Exception {
+    String url =
+        "/api/repository/v1/datasets/%s/data/%s/statistics/%s"
+            .formatted(datasetId, table, columnName);
+
+    String queryParams = "";
+
+    if (filter != null) {
+      queryParams += "?filter=%s".formatted(filter);
```

Comment:
Do we need to URL encode filter?

Reply:
Huh, that would make sense, but we have used this pattern across the rest of DataRepoFixtures and have several examples of it working without encoding. 
(ex: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/test/java/bio/terra/integration/AzureIntegrationTest.java#L596)

Reply:
Got it.  Not a huge deal since it's test code but maybe we'll want to update that at some point (totally fine to do that later)